packages/core/core-flows/src/definition/cart/workflows/complete-cart.ts

```diff
@@ -86,6 +87,13 @@ export const completeCartWorkflow = createWorkflow(
 
     linkOrderAndPaymentCollectionsStep(linkOrderPaymentCollection)
 
+    createRemoteLinkStep([
```

Comment:
we can use `paralellize` to call `linkOrderAndPaymentCollectionsStep` and `createRemoteLinkStep`